e2e-tests/gef/cypress/e2e/application/application-details.spec.js

```diff
@@ -168,12 +164,7 @@ context('Application Details Page', () => {
     });
 
     it('displays the correct submission type heading', () => {
-      applicationDetails
-        .mainHeading()
-        .invoke('text')
-        .then((text) => {
-          expect(text.trim()).to.equal('Application Details');
-        });
+      cy.assertText(applicationDetails.mainHeading(), 'Application Details');
```

Comment:
repeated twice, const for Application Details?

Reply:
I think this should be part of a wider ticket for content strings or similar.

Would not want to have a different "main heading content string/const" pattern in place specifically for this page. Also, this PR is purely about replacing cypress "text/trim/to.equal" text assertions with a new DRY command.